README.md

```diff
@@ -15,10 +15,14 @@ A Liferay portlet to update the sample-tracking information of selected samples.
 
 
 ## How to Install
-1. clone repository
-2. install maven
-3. run `mvn clean package`
-
+### Prerequisites
+1. A [Properly Configured Liferay Portal](https://github.com/qbicsoftware/qbic-setup-doc/blob/main/application_servers/liferay7.md) <!-- TODO: Where to link?-->
+2. Access to a running instance of the [sample-tracking service](https://github.com/qbicsoftware/sample-tracking-service)
+   
+3. An properly configured [openBIS instance](https://openbis.ch/) <!-- TODO where to link? -->
```

Comment:
what about the todos? Can they be solved

Reply:
nope there is no documentation as of now. The link to openbis partly fulfills the TODO for now but must be tackled again later on.

Reply:
So you will add the link as discussed in the daily?